### ISSUE TITLE
libSimTKmath issue

### ISSUE THREAD

**Issue by rmeftah** [opensim-org/opensim-core#3392]
Hello, 
I'm facing the following error.
When running my code, I'm having this error message :
```
Traceback (most recent call last):
  File "/home/etro/PycharmProjects/Mémoire/write_osim_model.py", line 1, in <module>
    import opensim as osim
  File "/home/etro/miniconda3/envs/memoire/lib/python3.7/site-packages/opensim-4.4.1-py3.7.egg/opensim/__init__.py", line 9, in <module>
    from .simbody import *
  File "/home/etro/miniconda3/envs/memoire/lib/python3.7/site-packages/opensim-4.4.1-py3.7.egg/opensim/simbody.py", line 13, in <module>
    from . import _simbody
ImportError: libSimTKmath.so.3.8: cannot open shared object file: No such file or directory
```

I try both installing through [conda](https://github.com/opensim-org/conda-opensim) and following the instruction available on the [OpenSim documentation](https://simtk-confluence.stanford.edu:8443/display/OpenSim/Scripting+in+Python) but it does not work.

Does someone know how to fix it ? 

thank you very much for your help.

**Comment by rmeftah** [comment]
I solved the issue by modifying the export path in the .bashrc file with the following command line: 
`export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/etro/opensim-core/sdk/lib:/home/etro/opensim-core/sdk/Simbody/lib
`
And I'm facing now the following error : 
 ```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/etro/opensim-core/sdk/Python/opensim/__init__.py", line 16, in <module>
    from .moco import *
  File "/home/etro/opensim-core/sdk/Python/opensim/moco.py", line 13, in <module>
    from . import _moco
ImportError: /home/etro/opensim-core/sdk/lib/libtropter.so: undefined symbol: _ZN8badoublecvRKdEv
```

Please help :sob:

**Comment by rmeftah** [comment]
I solved the issue by modifying the export path in the .bashrc file with the following command line: 
`export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/etro/opensim-core/sdk/lib:/home/etro/opensim-core/sdk/Simbody/lib
`
And I'm facing now the following error : 
 ```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/etro/opensim-core/sdk/Python/opensim/__init__.py", line 16, in <module>
    from .moco import *
  File "/home/etro/opensim-core/sdk/Python/opensim/moco.py", line 13, in <module>
    from . import _moco
ImportError: /home/etro/opensim-core/sdk/lib/libtropter.so: undefined symbol: _ZN8badoublecvRKdEv
```

I use Ubuntu 22.04 LTS.

Please help :sob:

**Comment by AlbertoCasasOrtiz** [comment]
Hello @rmeftah. I am sorry to hear that you are having this problem. This issue seems related to #2930. We have some problems linking moco libraries on Linux. Can you try using another OS (Mac, Windows)?

As noted in the [conda-opensim](https://github.com/opensim-org/conda-opensim) repository, the current conda package does not include moco, and you have to build it on your own. However, we are experiencing the same issues when building with moco.

I am personally working on the linking libraries problem issue. I will let you know once it is solved.

**Comment by AlbertoCasasOrtiz** [comment]
Hi @rmeftah, looks like this was solved in #2930. Could you test and close this issue if solved? Thank you!

**Comment by jenhicks** [comment]
Closing ... please re-open if issues are not resolved.
